sql/moz-fx-data-shared-prod/search_terms_derived/adm_weekly_aggregates_v1/query.sql

```diff
@@ -1,16 +1,17 @@
 SELECT
+  DATE_SUB(@submission_date, INTERVAL 6 DAY) AS start_date,
   @submission_date AS submission_date,
-  LOWER(search_query) AS query,
+  sanitized_query AS query,
```

Comment:
Note that we no longer need to use `LOWER` here because the `suggest_impression_sanitized_v2` logic has already stripped leading whitespace and lowercased results.